deploy/install/01-prerequisite/03-crd.yaml

```diff
@@ -1,201 +1,740 @@
 apiVersion: apiextensions.k8s.io/v1
 kind: CustomResourceDefinition
 metadata:
+  annotations:
+    controller-gen.kubebuilder.io/version: v0.7.0
+  creationTimestamp: null
   labels:
-    longhorn-manager: Engine
-  name: engines.longhorn.io
+    longhorn-manager: ""
```

Comment:
Notice that the value of this generated label is missed. Is there a way to fix that?

Reply:
I have no idea by leverage kustomize tool or other tool.

However,it's a label and the purpose is to filter the CRDs used by longhorn-manager. Without, the corresponding value, we still able to get the longhorn CRDs by "kubectl get crd -l longhorn-manager".

If we really care the label's value, the only way I could think now is hand-craft the label.

Reply:
I am fine with leaving the value empty. @innobead What do you think?

Reply:
It's still able to filter, so I am fine with that. But it's better to see if we have any internal code impacted by this or even documentation.